iogt/static/css/state/_s-article.scss

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
@Mitso Is this for the feature phone or smart phone styles? If it's for smart phone then perhaps it should be included in the `@media` query part.